src/openforms/registrations/api/filters.py

```diff
@@ -23,33 +25,54 @@ class ListInformatieObjectTypenQueryParamsSerializer(serializers.Serializer):
         label=_("ZGW API set"),
         required=False,
     )
+    objects_api_group = PrimaryKeyRelatedAsChoicesField(
+        queryset=ObjectsAPIGroupConfig.objects.all(),
+        help_text=_(
+            "The primary key of the Objects API group to use. The informatieobjecttypen from the Catalogi API "
+            "in this group will be returned."
+        ),
+        label=_("Objects API group"),
+        required=False,
+    )
     registration_backend = serializers.ChoiceField(
         help_text=_("The ID of the registration backend to use."),
         label=_("Registration backend ID"),
-        required=False,
         choices=[],
     )
 
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        registration_backend = attrs["registration_backend"]
+        if registration_backend == "objects_api" and "objects_api_group" not in attrs:
```

Comment:
You mean define a `PLUGIN_IDENTIFIER` constant?

Reply:
no, because the `openforms.registrations.api` package should not be aware of individual/specific plugins at all, ideally all interactions should be done in a generic way through the plugin registry.

That's a shit ton of work that's out of scope, so a TODO for now is fine so we at least show this is a known bad pattern to future people touching this code.